packages/ui/src/elements/ListControls/index.tsx

```diff
@@ -193,6 +197,25 @@ export const ListControls: React.FC<ListControlsProps> = (props) => {
                 {t('general:sort')}
               </Pill>
             )}
+            {listControlsMenu != null && listControlsMenu !== false && (
+              <Popup
+                button={<Dots />}
+                className={`${baseClass}__popup`}
+                horizontalAlign="right"
+                size="large"
+                verticalAlign="bottom"
+              >
+                <PopupList.ButtonGroup>
+                  {Array.isArray(listControlsMenu) ? (
```

Comment:
Do not need the ternary, always assume an array.

Reply:
Updated